CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing
+
+Anyone is welcome to help with Firefox Accounts. Feel free to get in touch with other community members on IRC, the
+mailing list or through issues here on GitHub.
+
+- IRC: `#fxa` on `irc.mozilla.org`
+- Mailing list: <https://mail.mozilla.org/listinfo/dev-fxacct>
+- and of course, [the issues list](https://github.com/mozilla/fxa-basket-proxy/issues)
+
+## Bug Reports ##
+
+You can file issues here on GitHub. Please try to include as much information as you can and under what conditions
+you saw the issue.
+
+## Sending Pull Requests ##
+
+Patches should be submitted as pull requests (PR).
+
+Before submitting a PR:
+- Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviewed.
+  - Run `grunt lint` to make sure your code passes linting.
+  - Run `npm test` to make sure all tests still pass.
+- Your patch should include new tests that cover your changes. It is your and your reviewer's responsibility to ensure your patch includes adequate tests.
+
+When submitting a PR:
+- You agree to license your code under the project's open source license ([MPL 2.0](/LICENSE)).
+- Base your branch off the current `master` (see below for an example workflow).
+- Add both your code and new tests if relevant.
+- Run `grunt eslint` and `npm test` to make sure your code passes linting and tests.
+- Please do not include merge commits in pull requests; include only commits with the new relevant code.
+
+After your PR is merged:
+- Add yourself to the [AUTHORS](/AUTHORS) file so we can publicly recognize your contribution.
+
+See the main [README.md](/README.md) for information on prerequisites, installing, running and testing.
+
+## Code Review ##
+
+This project is production Mozilla code and subject to our [engineering practices and quality standards](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Committing_Rules_and_Responsibilities). Every patch must be peer reviewed. This project is part of the [Firefox Accounts module](https://wiki.mozilla.org/Modules/Other#Firefox_Accounts), and your patch must be reviewed by one of the listed module owners or peers.
+
+## Example Workflow ##
```

Comment:
The example workflow section is awesome documentation and should be added to the content server and any other repo that could benefit from it.